queries/qmljs/highlights.scm

```diff
@@ -1,6 +1,13 @@
 ; temporarily removed the ecma inherit due to it breaking
 "pragma" @keyword.import
 
+(ui_import
+  "import" @keyword.import)
```

Comment:
Can't this just be `"import" @keyword.import`

Reply:
Emm. I just worry that maybe it will find other place..